packages/astro/src/@types/astro.ts

```diff
@@ -452,6 +453,27 @@ export interface AstroUserConfig {
 	 */
 	cacheDir?: string;
 
+	
+
+	/**
+	 * @docs
+	 * @name redirects
```

Comment:
I’m noticing that the `image.service` option includes “(Experimental)” after the name and mentions it in the description too: 

![Image service docs excerpt](https://github.com/withastro/astro/assets/357379/3aa3a4b1-7ad2-4cc3-9ec5-a07be750e5ea)

Is that something we want to do for redirects too so it’s clear you need to enable the flag to get these settings?